src/lucky/mount_component.cr

```diff
@@ -6,47 +6,10 @@ module Lucky::MountComponent
   # starts and ends.
   #
   # ```
-  # mount MyComponent.new
+  # mount(MyComponent)
+  # mount(MyComponent, with_args: 123)
   # ```
-  @[Deprecated("Use `#m` instead. Example: m(MyComponent, arg1: 123)")]
-  def mount(component : Lucky::BaseComponent) : Nil
-    print_component_comment(component.class) do
-      component.view(view).render
-    end
-  end
-
-  # Appends the `component` to the view. Takes a block, and yields the
-  # args passed to the component.
-  #
-  # When `Lucky::HTMLPage.settings.render_component_comments` is
-  # set to `true`, it will render HTML comments showing where the component
-  # starts and ends.
-  #
-  # ```
-  # mount MyComponent.new("jane") do |name|
-  #   text name.upcase
-  # end
-  # ```
-  @[Deprecated("Use `#m` instead. Example: m(MyComponent, arg1: 123) do/end")]
-  def mount(component : Lucky::BaseComponent) : Nil
```

Comment:
Can you add a deprecated method for `m` that tells people to use `mount` instead?

Reply:
@paulcsmith For sure! I'll submit that later this week.